functions/src/auto-importer.ts

```diff
@@ -255,15 +267,27 @@ export const syncToS3AfterSyncDocWritten = functions.firestore
           const needSyncMoreRecentThanStartSync = data.need_sync && (!data.start_sync || (data.need_sync > data.start_sync));
 
           if (data.need_sync && needSyncMoreRecentThanDidSync && needSyncMoreRecentThanStartSync) {
-            const syncDocRef = getAnswerSyncCollection(context.params.syncSource).doc(context.params.runKey);
+            const syncDocRef = change.after.ref;
```

Comment:
This is nice. I was wondering if you could optimize it like this.

Reply:
Deeply buried in documentation....